internal/installcfg/installcfg.go

```diff
@@ -229,7 +255,7 @@ func applyConfigOverrides(overrides string, cfg *InstallerConfigBaremetal) error
 	return nil
 }
 
-func GetInstallConfig(log logrus.FieldLogger, cluster *common.Cluster, addRhCa bool, ca string) ([]byte, error) {
+func getInstallConfigStruct(log logrus.FieldLogger, cluster *common.Cluster, addRhCa bool, ca string) (*InstallerConfigBaremetal, error) {
```

Comment:
Don't love this name. Any suggestions?

Reply:
IMO this name should be just `getInstallConfig` while the other method which is called `GetInstallConfig` should be renamed to `MarshalInstallConfig`. That will create a bigger PR and not sure if that worths the effort. 

Simpler solution would be calling this one `getInstallConfig` which can co-live with the exported one `GetInstallConfig`